pkg/proxy/nftables/proxier.go

```diff
@@ -63,6 +64,16 @@ const (
 	// so they don't need any "kube-" or "kube-proxy-" prefix of their own.
 	kubeProxyTable = "kube-proxy"
 
+	// base chains
+	kubeFilterPreRouting     = "filter-prerouting"
+	kubeFilterInput          = "filter-input"
+	kubeFilterForward        = "filter-forward"
+	kubeFilterOutput         = "filter-output"
+	KubeFilterOutputPostDNAT = "filter-output-post-dnat"
+	kubeNATPreRouting        = "nat-prerouting"
+	kubeNATOutput            = "nat-output"
+	kubeNATPostRouting       = "nat-postrouting"
```

Comment:
For consistency with the other chains, these constants should all have `Chain` at the end of their name (eg `kubeFilterInputChain`)

(Also, I'm realizing that the reason the existing constants have names starting with `kube` is because in the iptables proxy, the actual chain names all started with `KUBE` too... I probably should have renamed them in nftables... You do not need to fix this, but you can if you want. If you do, you should fix the existing constants first as a separate commit.)